mmdet/optimizer/registry.py

```diff
@@ -0,0 +1,3 @@
+from mmdet.utils import Registry
+
+OPTIMIZERS = Registry('optimizer')
```

Comment:
We may register all built-in optimizers of PyTorch here to simplify the builder.